### ISSUE TITLE
Allow exclusion of certain file types/extensions

### ISSUE THREAD

**Issue by lckarssen** [pulsejet/memories#419]
My camera saves its photos in both raw (`.RAF`) and jpeg format. Since I store both files together (in the same folder) these photos show up twice in e.g. the timeline. Would it be possible to have a setting where we can exclude certain file types or file extensions? Or, maybe a bit fancier, to exclude a certain file type only if a photo/file with the same basename exists?

**Comment by pulsejet** [comment]
Can you provide some samples, i.e. the RAF and JPEG for the same photo?

**Comment by lckarssen** [comment]
Some more info: when I filed this bug,  I had the [camera raw preview app](https://apps.nextcloud.com/apps/camerarawpreviews) installed. I recently removed the external storage on which most of my photos are stored (they subsequently disappeared from Memories, obviously). I then disabled the raw preview app, cleared all previews and re-added the external storage. Now I no longer see the RAW files in Memories, but (again obviously) also don't get previews in Files any more. 

From #537 I learned about #152. This issue can be seen a duplicate of it. I would be (very) happy with the solution proposed in https://github.com/pulsejet/memories/issues/152#issuecomment-1354592883 of stacking the JPG and RAW. 

Example files:
- [JPG file](https://cloud.karssen.org/index.php/s/DP5rSc4EZw2BGrF)
- [Fuji RAW file (`.RAF`)](https://cloud.karssen.org/index.php/s/SsNpcoE8MfCP2Z9)
